src/app/badge/badge.component.scss

```diff
@@ -0,0 +1,15 @@
+@import "../styles/main";
+
+.badge-name {
+  color: $yellow;
+  font-weight: $bold;
+  font-size: $font-size-med;
+}
+
+.body {
+  text-align: center;
+}
+
+img {
+  height: 10em;
+}
```

Comment:
new line

Reply:
@Raxerz I checked it and it has a new line. Please check again.

Reply:
Its not there @lunayach